src/frontend/apps/impress/src/features/docs/doc-header/components/DocToolBox.tsx

```diff
@@ -29,17 +37,90 @@ interface DocToolBoxProps {
 
 export const DocToolBox = ({ doc, versionId }: DocToolBoxProps) => {
   const { t } = useTranslation();
+  const { spacingsTokens, colorsTokens } = useCunninghamTheme();
+
+  const spacings = spacingsTokens();
+  const colors = colorsTokens();
+
   const [isModalShareOpen, setIsModalShareOpen] = useState(false);
   const [isModalRemoveOpen, setIsModalRemoveOpen] = useState(false);
   const [isModalPDFOpen, setIsModalPDFOpen] = useState(false);
-  const [isDropOpen, setIsDropOpen] = useState(false);
+
   const { setIsPanelOpen, setIsPanelTableContentOpen } = usePanelEditorStore();
   const [isModalVersionOpen, setIsModalVersionOpen] = useState(false);
   const { isSmallMobile } = useResponsiveStore();
   const { authenticated } = useAuthStore();
   const { editor } = useEditorStore();
   const { toast } = useToastProvider();
 
+  const options: DropdownMenuOption[] = [
+    ...(isSmallMobile
```

Comment:
It will merge both of them:
![image](https://github.com/user-attachments/assets/813241ea-8f8f-481b-8b4a-59aa3aa93226)